<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExplainWindow</class>
 <widget class="QMainWindow" name="ExplainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Объяснение</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ruleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Правила</string>
    </property>
   </widget>
   <widget class="QPushButton" name="expandRulesButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>261</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Раскрыть все</string>
    </property>
   </widget>
   <widget class="QTreeView" name="rulesTree">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="varLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Переменные</string>
    </property>
   </widget>
   <widget class="QTableView" name="varsView">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>321</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
